packages/conventional-gitmoji/src/config/rewrites.ts

```diff
@@ -8,63 +8,69 @@ type RewritesProps = {
   releaseNotes?: boolean | null
   semver?: 'breaking' | 'feature' | 'fix' | 'major' | 'minor' | 'patch' | null
   to: string
+  title?: string
 }
 
 /* eslint-disable sort-keys */
 const rewrites: RewritesProps[] = [
   { from: 'art', to: 'style' },
-  { from: 'zap', to: 'perf' },
+  { from: 'zap', to: 'perf', title: 'Performance' },
```

Comment:
We can probably remove `title` completely as we started down this path but then ultimately decided to forego it in the release notes in favor of the description/section.